src/main/resources/log4j2.properties

```diff
@@ -3,7 +3,7 @@ appender=console
 appender.console.type = Console
 appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = {"timestamp": "%d{YYYY-MM-dd HH:mm:ss.SSS}", "level" : "%-5p", "thread" : "%T", "logger" : "%c{1}:%L", "message" : "%msg", "context" : default} 
+appender.console.layout.pattern = {"timestamp": "%d{YYYY-MM-dd HH:mm:ss.SSS}", "level" : "%-5p", "thread" : "%T", "logger" : "%c{1}:%L", "message" : "%msg", "context" : default%n} 
```

Comment:
%n It's in wrong place you wil get something 
**{logs}
}**
%n should be at end of log ( ... "context" : default}%n )

Reply:
Sorry, I didn't see that in my test